tests/model_registry/rbac/utils.py

```diff
@@ -0,0 +1,149 @@
+from typing import Any, Dict, Generator
+from pyhelper_utils.shell import run_command
+from utilities.constants import Protocols
+import logging
+from contextlib import contextmanager
+from ocp_resources.namespace import Namespace
+from model_registry import ModelRegistry as ModelRegistryClient
+from tests.model_registry.utils import get_endpoint_from_mr_service, get_mr_service_by_label
+from ocp_resources.model_registry import ModelRegistry
+from utilities.infra import get_openshift_token
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.group import Group
+
+LOGGER = logging.getLogger(__name__)
+
+
+def build_mr_client_args(rest_endpoint: str, token: str, author: str) -> Dict[str, Any]:
+    """
+    Builds arguments for ModelRegistryClient based on REST endpoint and token.
+
+    Args:
+        rest_endpoint: The REST endpoint of the Model Registry instance.
+        token: The token for the user.
+        author: The author of the request.
+
+    Returns:
+        A dictionary of arguments for ModelRegistryClient.
+
+    Note: Uses is_secure=False for testing purposes.
+    """
+    server, port = rest_endpoint.split(":")
+    return {
+        "server_address": f"{Protocols.HTTPS}://{server}",
+        "port": port,
+        "user_token": token,
+        "is_secure": False,
+        "author": author,
+    }
+
+
+@contextmanager
+def switch_context(context_name: str) -> Generator[None, None, None]:
```

Comment:
Can you please rename to `switch_user_context` or `switch_openshift_user_context`?

Reply:
It is not yet addressed.